packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -28,14 +30,21 @@ const MeetingInfoUtil = {};
 MeetingInfoUtil.isMeetingLink = (value) => {
   // eslint-disable-next-line no-warning-comments
   // TODO: lets remove regex from this equation and user URI matchers and such
-  const regex = /(.*).webex.com\/(?:meet|m)\/(.*)/;
-  const m = regex.exec(value);
+  const m = MEETING_LINK_SIP_REGEX.exec(value);
```

Comment:
Can we _not_use a regex please? As the TODO says, we should just parse the url that way we can assert on various elements of the uri. 

```
const meetingLink = new URL(url);
if (meetingLink.hostname === 'foo') { }
```

etc. Is there value to the above?

Reply:
yeah i mean i want to find a more complete solution than URL because we'd still need a regex to ensure that *.webex.com is a a correct regex
and that meet or m or join appears
and that we can extract the user from /meet/<user>

I agree we should get away from regex, i added that TODO, we just didn't have cycles to work on it at the time.

Reply:
and we need to ensure not to include the www for converting links to sip URI

Reply:
I would rather do this now than punt on it. Otherwise the regex will live forever. I can help contribute the code.